docs/main_text.md

```diff
@@ -312,8 +313,8 @@ Note that only gene expression matrices that we are able to successfully quantil
 #### Limitations of quantile normalization across platforms with many zeroes
 
 Quantile normalization is a strategy that can address many technical effects, generally at the cost of retaining certain sources of biological variability.
-However, in cases where there are many ties that are different between samples, the transformation can produce outputs with different distributions.
-This arises in data processed by refine.bio when RNA-seq and microarray data are combined.
+We use a single reference distribution per organism, generated from the Affymetrix microarray platform with the largest number of samples we were able to process from raw (see [_Reference distribution_](#reference=distribution)).
+In cases where there are many ties that are different between samples, such as when RNA-seq data and microarray data are combined into a single dataset or matrix, the transformation can produce outputs with different distributions.
```

Comment:
I assume you are referring to within-sample ties here? How often does that come up in microarray data? I would assume it is pretty rare, and that the problem mostly occurs with RNA-seq data? 

Second, how different are the distributions we end up with? I suspect not that different? 

I'm putting a suggestion below that incorporates my understanding, but it might not be correct...

```suggestion
In cases where the raw data contains many ties within samples (and the ties are different between samples), the transformation can produce outputs with somewhat different distributions for each sample.
This arises most often in when RNA-seq and microarray data are combined into a single dataset or matrix.
```

Reply:
> I would assume it is pretty rare, and that the problem mostly occurs with RNA-seq data?

Yep. With the caveat that I have not looked at every single microarray sample we've ever processed 😄  The problem that motivates this section is all the zeroes (zeros?) in RNA-seq.

> Second, how different are the distributions we end up with? I suspect not that different?

Different enough to fail the testing scenario where you do a KS test on _all_ values including the lowly expressed ones. It's not different if you were just to look at the top half of the expression values (which is what I would probably recommend working with in many downstream analyses when you aggregate by species). Agree with "somewhat different" characterization.

Reply:
Took another pass using some of your wording in https://github.com/AlexsLemonade/refinebio-docs/pull/143/commits/1cdd76e9f509c69465610dad8e860efb5df07ea4, let me know what you think!